t/api/01-workers.t

```diff
@@ -147,5 +148,25 @@ $ret = $t->post_ok('/api/v1/workers', form => $worker_caps);
 is($ret->tx->res->code,       200, "register new worker");
 is($ret->tx->res->json->{id}, 3,   "new worker id is 3");
 
+subtest 'delete offline worker' => sub {
+    my $offline_worker_id = 9;
+    my $database_workers  = $t->app->schema->resultset("Workers");
+    $database_workers->create(
+        {
+            id        => $offline_worker_id,
+            host      => 'offline_test',
+            instance  => 5,
+            t_updated => time2str('%Y-%m-%d %H:%M:%S', time - 1200, 'UTC'),
+        });
+
+    $ret = $t->delete_ok("/api/v1/workers/$offline_worker_id");
+    is($ret->tx->res->code, 200, "delete offline worker successfully.");
```

Comment:
You can simplify this by using `status_is(200)`

Reply:
Also `$ret` is simply `$t` again here.

Reply:
I'd consider leaving this out of scope for the context of this PR given that's the style of the whole test case. And this would actually make sense to update in a separate PR with no logical changes but just simplifying it.